source/educators/quickstarts/build_a_course.rst

```diff
@@ -1,20 +1,17 @@
 Quick Start: Build a Course
 ####################################
 
-.. sidebar:: Create Course Video Demo
-
-  |Video Sidebar Click|
-
-  Video TBP
-
 .. contents:: Steps
   :local:
   :class: no-bullets
 
 
+Follow the steps below to build your first course. By the end, you will have a fully functional, minimal course, which you can build on as you learn all of the |Platform| options. |Image Instructions|
 
 
-Follow the steps below to build your first course. By the end, you will have a fully functional, minimal course, which you can build on as you learn all of the |Platform| options. |Image Instructions|
+.. collapse:: Building a Course: Video Demonstration
```

Comment:
Why did you put it here instead of in the sidebar?

Reply:
@feanil I tried.  cannot get the thumbnail to work in the sidebar.  Ideally, I wanted to use lightbox and have the video pop up, but it's not compatible.  I can keep investigating that if you think it's worth it.

Reply:
No, I think this is fine, but I wonder if it could be made more visible.  I could imagine a lot of people jumping past the video if it's collapsed.

Reply:
@feanil I removed the collapse